### PR TITLE
move presets into provider

### PR DESCRIPTION
**Pull request by tkraak**

### CODE REVIEW

File: src/angular-dayparts.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
 angular.module('angular-dayparts', [])
-.directive('angularDayparts', ['$window', '$document', '$timeout', function ($window, $document, $timeout) {
+  .provider('angularDaypartsConfig', function () {
+    this.presetItems = [
+      {"value":"week", "label":"All hours and days"},
+      {"value":"weekend", "label":"Weekends (Sat-Sun)"},
+      {"value":"weekdays", "label":"Weekdays (Mon-Fri)"},
+      {"value":"businessHours", "label":"Business Hours (Mon-Fri, 9am-5pm)"},
+      {"value":"eveningHours", "label":"Evenings (6pm-12pm)"},
+      {"value":"custom", "label":"Custom", disabled: true}
```
Comment (FrenchDilettante) [inline src/angular-dayparts.js:9]: Should it be disabled by default?

File: src/template.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="dayparts">
-  <select ng-model="selectedPresetItem" ng-options="item.value as item.label disable when item.disabled for item in presetItems" ng-change="selectedPresetItemChanged()"></select>
+  <select class="form-control" ng-model="selectedPresetItem" ng-options="item.value as item.label disable when item.disabled for item in presetItems" ng-change="selectedPresetItemChanged()"></select>
```
Comment (FrenchDilettante) [inline src/template.html]: This is not a related change to presets. Although, it could be an additional property on the config provider :)

**Review by FrenchDilettante — CHANGES_REQUESTED** [review]

File: src/angular-dayparts.js (hunk 1 above)
Comment (tkraak) [reply src/angular-dayparts.js:9]: The demo in this repo is set up for AngularJS `1.6.1`, which means `disabled: true` should be the default ⚡️ .

File: src/template.html (hunk 2 above)
Comment (tkraak) [reply src/template.html]: I moved this into the config provider, thx!

**Review by FrenchDilettante — APPROVED** [review]

**Pull request merged**